bucket/libgen-desktop.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "1.4.1",
+    "description": "Desktop for Library Gensis",
+    "homepage": "https://github.com/libgenapps/LibgenDesktop",
+    "license": "Unknown",
+    "notes": [
+        "Require Windows Vista SP2 or later and .NET Framework 4.5 (Included in Windows 8 or later by default) or higher to run.",
+        "See also https://wiki.mhut.org/software:libgen_desktop for details."
+    ],
```

Comment:
```suggestion
```

This section is not required. Scoop is only supported on Windows 7 and above (which already have this).